changelog/issue-4998.md

```diff
@@ -0,0 +1,6 @@
+audience: users
+level: patch
```

Comment:
I'm torn between this being patch and being major.

Reply:
I'm not sure how much does this change break for existing integrations.

Reply:
The workaround for this bug should keep working after this.

Also in general, it's possible for users to depend on the buggy behaviour of almost any bug, so personally I don't think it should be major.